assets/src/components/groupedAutocomplete.tsx

```diff
@@ -672,6 +677,9 @@ interface GroupedAutocompleteFromSearchTextResultsProps
    *   type deduction.
    */
   onSelectVehicleOption: (selectedOption: Vehicle | Ghost) => void
+
+  onSelectedLocationId: (selectedPlaceId: string) => void
+  onSelectedLocationText: (selectedLocationText: string) => void
```

Comment:
suggestion: Do you have any thoughts on how to document these? I'm struggling to come up with comments that describe non-obvious parts of these callbacks.

Here's a rough approximation I came up with.
```suggestion
  /**
   * Fired when a autocomplete option with a PlaceId is selected.
   * @param selectedPlaceId Suggested PlaceId.
   */
  onSelectedLocationId: (selectedPlaceId: string) => void
  /**
   * Fired when a autocomplete option without a PlaceId is selected.
   * @param selectedLocationText Suggested Location Search Text.
   */
  onSelectedLocationText: (selectedLocationText: string) => void
```

Reply:
Yeah I like that!